ios/REANodesManager.m

```diff
@@ -553,4 +595,36 @@ - (NSString *)obtainProp:(nonnull NSNumber *)viewTag propName:(nonnull NSString
   return result;
 }
 
+- (void)tryToFlushUpdateBuffer
```

Comment:
I'd rename it to `maybeFlushUpdateBuffer` as `try` suggest that we are ok with it failing